### PR TITLE
blockchain using python

### PR DESCRIPTION
**Pull request by sonika-shah**

### CODE REVIEW

File: Submissions/sonika-shah/Assignment1/1.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+class block():
+	def __init__(self, timestamp, data, previousHash = ''):
+		self.timestamp = timestamp
+		self.data = data
+		self.previousHash = previousHash
+		self.hash = self.calculateHash()
+
+	def calculateHash(self):
+		return sha256((str(self.timestamp) + str(self.data) + str(self.previousHash).encode()).hexdigest())
```
Comment (johnsoncarl) [inline Submissions/sonika-shah/Assignment1/1.py]: please check the pranatheses here.

**Review by johnsoncarl — CHANGES_REQUESTED** [review]
Please remove the inconsistencies

**Comment by sonika-shah** [comment]
removed the inconsistencies

**Review by johnsoncarl — APPROVED** [review]
cool... Good Work 👍

**Pull request merged**